README.md

```diff
@@ -487,6 +488,37 @@ castNotNull: 3
 
 ----
 
+
+##### safeCast
+
+Adapt this `Flow<*>` to be a `Flow<R?>`.
+
+At the collection time, converts a Flow<*> to a Flow<R?>, if not consistent with R, returns null.
```

Comment:
Same as docs in kotlin 🙏
> Adapt this `Flow<*>` to be a `Flow<R?>`.
At the collection time, if this Flow has any value that is not an instance of R, `null` will be emitted